### ISSUE TITLE
Add tumor purity estimate to pipeline

### ISSUE THREAD

**Issue by ihodes** [hammerlab/epidisco#56]

**Comment by armish** [comment]
I am planning to work on this week. OK with you @ihodes?

**Comment by ihodes** [comment]
Definitely! It's not a blocker for PGV001, but would be awesome to have nonetheless.

**Comment by ihodes** [comment]
A nice QC measure at the very least for the current trial.

**Comment by armish** [comment]
yeah, we also want this for the RCC stuff - so would be great to have this in epidisco to generalize.

No promises on a robust QC metric, but I am proud of our in-house purity estimates from VCFs ;)

**Comment by ihodes** [comment]
@armish still on your radar?

**Comment by armish** [comment]
Yep! Will be bundling this with the planned QC improvements: https://github.com/hammerlab/biokepi/pull/381

**Comment by ihodes** [comment]
Awesome, thank you!
